src/oncall/ui/static/js/oncall.js

```diff
@@ -92,7 +96,11 @@ var oncall = {
       var data = JSON.parse(data),
           token = data.csrf_token;
 
-      $.ajaxSetup({headers: {'X-CSRF-TOKEN': token}});
+      $.ajaxPrefilter(function(options, originalOptions, jqXHR) {
```

Comment:
im a bit confused as to what this block is doing. is there any reason to re-set the ajaxprefilter on error?